server/routes/index.js

```diff
@@ -0,0 +1,44 @@
+// Controllers
+const userController = require('../controllers').users;
+
+module.exports = (app, passport) => {
+    // Setup a default catch-all route that sends back a welcome message
+    app.get('/', (req, res) => {
+        return res.status(200).send({
+            message: 'CHEMICAL-X',
+        });
+    });
+
+    app.get('/users', isLoggedIn, userController.list);
```

Comment:
I think this is ok for now but in the future, we probably don't want users to be able to see all users (we can talk about this later).